package.json

```diff
@@ -50,9 +50,9 @@
     "cli-unit": "yarn unit-cli",
     "viewer-unit": "yarn unit-viewer",
     "watch": "yarn unit-core --watch",
-    "unit:cicoverage": "yarn c8 --all yarn unit:ci && yarn c8 report --reporter text-lcov > unit-coverage.lcov",
+    "unit:cicoverage": "yarn c8 --all yarn unit:ci",
     "coverage": "yarn unit:cicoverage && c8 report --reporter html",
-    "smoke:cicoverage": "yarn c8 yarn smoke -j=1 --retries=2 && yarn c8 report --reporter text-lcov > smoke-coverage.lcov",
+    "smoke:cicoverage": "yarn c8 yarn smoke -j=1 --retries=2",
```

Comment:
should we just remove the -j and --retries flags here? better defined in the yml anyhow

Reply:
I thought about that but then didn't do it because what's the point of having a `cicoverage` npm script just to call `yarn c8 yarn smoke` since it's not ci-specific and it only really works with the extra flags? Well...there is no point, so let's move those to the yml and delete this :)